annotations/annotations_rw_test.go

```diff
@@ -264,21 +281,27 @@ func TestRWHappyGTG(t *testing.T) {
 func TestRWHTTPRequestErrorGTG(t *testing.T) {
 	rw := NewRW(testClient, ":#")
 	err := rw.GTG()
-	assert.EqualError(t, err, "gtg HTTP request error: parse :: missing protocol scheme")
+	e := errors.Unwrap(err)
+	var urlError *url.Error
+	assert.True(t, errors.As(err, &ErrGTGRequest))
+	assert.True(t, errors.As(e, &urlError))
 }
 
 func TestRWHTTPCallErrorGTG(t *testing.T) {
 	rw := NewRW(testClient, "")
 	err := rw.GTG()
-	assert.EqualError(t, err, "gtg HTTP call error: Get /__gtg: unsupported protocol scheme \"\"")
+	e := errors.Unwrap(err)
+	var urlError *url.Error
+	assert.True(t, errors.As(err, &ErrGTGCall))
```

Comment:
Here you change the value of `ErrGTGCall` and test nothing. I would suggest to wrap `ErrGTGCall` and not `url.Error` and test with `errors.Is`